src/Form.vue

```diff
@@ -448,12 +451,20 @@ export default {
 
         // default
         if (question.default !== undefined) {
-          let _default;
-          if (typeof question.default !== "function") {
-            _default = question.default;
-          }
-          this.$set(question, "_default", _default);
-        }
+	  let _default;
+	  if (typeof question.default !== "function") {
+	     if (question.__origAnswer === undefined) {
+		_default = question.default;
+	     } else {
+		_default = question.__origAnswer; 
+	     }
+	  }
+	  this.$set(question, "_default", _default);
+	}
+
+	if (question._default === undefined) {
+	   this.$set(question, "_default", question.__origAnswer);
+	}
```

Comment:
} else {
	if (question._default === undefined) {
	   this.$set(question, "_default", question.__origAnswer);
	}
}

Reply:
Done